### PR TITLE
1password,[ab]*: ensure event.kind is correctly set for pipeline errors

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

Modify 1password, akamai, atlassian_*, auditd*, auth0, azure_frontdoor, barracuda_cloudgen_firewall, bitdefender, bluecoat, box_events to correctly set `event.kind` for pipeline errors and ensure `error.message` is an array.

Also fixes changelog links for ti_cif3, ti_otx. and ti_threatq.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- For #6582

### CODE REVIEW

File: packages/ti_cif3/changelog.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@
   changes:
     - description: Honor `preserve_original_event` setting.
       type: enhancement
-      link: https://github.com/elastic/integrations/pull/1
+      link: https://github.com/elastic/integrations/pull/5205
```
Comment (kgeller) [inline packages/ti_cif3/changelog.yml:21]: Revert this?

File: packages/ti_otx/changelog.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@
   changes:
     - description: Honor `preserve_original_event` setting.
       type: enhancement
-      link: https://github.com/elastic/integrations/pull/1
+      link: https://github.com/elastic/integrations/pull/5205
```
Comment (kgeller) [inline packages/ti_otx/changelog.yml:21]: Also revert?

File: packages/ti_threatq/changelog.yml [hunk 3]
```diff
@@ -18,7 +18,7 @@
   changes:
     - description: Honor `preserve_original_event` setting.
       type: bugfix
-      link: https://github.com/elastic/integrations/pull/1
+      link: https://github.com/elastic/integrations/pull/5205
```
Comment (kgeller) [inline packages/ti_threatq/changelog.yml:21]: Revert?

File: packages/ti_cif3/changelog.yml (hunk 1 above)
Comment (efd6) [reply packages/ti_cif3/changelog.yml:21]: This was a drive-by fix for a change link that was missed in a previous PR. Same for the other links to 5205

**Review by kgeller — APPROVED** [review]

**Pull request merged**